src/OrbitCore/SymbolHelper.h

```diff
@@ -33,6 +33,7 @@ class SymbolHelper {
 
   [[nodiscard]] fs::path GenerateCachedFileName(const fs::path& file_path) const;
 
+  [[nodiscard]] static bool IsMatchingDebugInfoFile(const fs::path& file_path, uint32_t checksum);
```

Comment:
probably did not mean to make this static

Reply:
Hmm why not? It does not need access to any member variables. I could also make it a free function if you prefer that?

Reply:
nevermind, missed the part where you moved it to SymbolHelper class.